packages/odyssey/src/scss/base/_elements.scss

```diff
@@ -0,0 +1,83 @@
+// Note: This file contians empty, commented out style definitons. This means they were not overriden in anyway, but accounted for in documentation.
+
+$blockquote-border-left: 3px solid cv('gray', '200');
+$ins-background: cv('success', 'lightest');
+$del-background: cv('danger', 'lightest');
+$mark-background: cv('warning', 'lightest');
+
+blockquote {
+    border-left: $blockquote-border-left;
+    padding: 0 $base-spacing;
+}
+
+// cite {}
+
+del {
+    background: $del-background;
+    display: inline-block;
+}
+
+del::before, 
+del::after {
+  clip-path: inset(100%);
+  clip: rect(1px, 1px, 1px, 1px);
+  height: 1px;
+  overflow: hidden;
+  position: absolute;
+  white-space: nowrap;
+  width: 1px;
+}
+
+del::before {
+  content: " [deletion start] ";
```

Comment:
This won't work with 18n, sadly

Reply:
We could do something like `content: attr(data-a11y)` instead to ensure these get announced.

Also, is there something in the visually hidden mixin that doesn't work for these?

Reply:
@edburyenegren-okta is `data-a11y` a convention we currently use? If not, it should be. 

Will fix.